quickwit-cli/src/main.rs

```diff
@@ -79,6 +79,155 @@ fn setup_logging_and_tracing(level: Level) -> anyhow::Result<()> {
     Ok(())
 }
 
+#[allow(dead_code)]
```

Comment:
We can't merge that. This code should be probably put somewhere else (in a script?)

Reply:
I moved it to its own binary